### PR TITLE
📺 Rework channel log rendering to always start from JSON

### PR DESCRIPTION
**Pull request by rowanseymour**
Also changes anonymization of logs: staff users will see anonymized by default and have to eplicitly break out of that with a url parameter (that we'll probably turn into something in the UI).

Making it easier to switch to loading channel logs from JSON blobs in S3.

### CODE REVIEW

File: temba/channels/models.py [hunk 1]
```diff
@@ -958,78 +960,56 @@ class ChannelLog(models.Model):
...
 
-    def _get_display_value(self, user, original, urn, redact_keys=(), redact_values=()):
-        """
-        Get a part of the log which may or may not have to be redacted to hide sensitive information in anon orgs
-        """
-
-        from temba.request_logs.models import HTTPLog
-
-        for secret_val in redact_values:
-            original = redact.text(original, secret_val, HTTPLog.REDACT_MASK)
```
Comment (rowanseymour) [inline temba/channels/models.py:969]: courier is doing this now and the only thing in `redact_values` is our Whatsapp system creds.. am on the fence about re-checking here vs assuming courier and its tests are good

Comment (rowanseymour) [reply temba/channels/models.py:969]: I've added code now that asserts that none of our credential values are in the returned json

File: temba/channels/types/instagram/type.py [hunk 2]
```diff
@@ -39,6 +40,8 @@ class InstagramType(ChannelType):
     max_length = 2000
     free_sending = True
 
+    redact_values = (settings.FACEBOOK_APPLICATION_SECRET, settings.FACEBOOK_WEBHOOK_SECRET)
```
Comment (norkans7) [inline temba/channels/types/instagram/type.py:43]: We need similar for `settings.WHATSAPP_ADMIN_SYSTEM_USER_TOKEN` and `settings.WHATSAPP_ADMIN_SYSTEM_USER_ID`?

Comment (rowanseymour) [reply temba/channels/types/instagram/type.py:43]: The whatsapp type already has `redact_values` set.. note that this is just checking that courier did its job

**Pull request merged**